packages/webapp/__tests__/PostPage.tsx

```diff
@@ -226,6 +226,34 @@ it('should show login on upvote click', async () => {
   expect(showLogin).toBeCalledTimes(1);
 });
 
+it('should check meta tag', async () => {
+  const seo = getSeoDescription(
+    createPostMock({
+      summary: 'Test summary',
+    }).result.data.post,
+  );
+
+  expect(seo).toEqual('Test summary');
+});
+
+it('should check meta tag', async () => {
+  const seo = getSeoDescription(
+    createPostMock({
+      description: 'Test description',
+    }).result.data.post,
+  );
+
+  expect(seo).toEqual('Test description');
+});
+
```

Comment:
Also we should probably add one with both summary and description to make sure it favours the summary.